hardhat.config.ts

```diff
@@ -0,0 +1,32 @@
+import { HardhatUserConfig, task } from "hardhat/config";
+import "@nomiclabs/hardhat-ethers";
+
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+task("accounts", "Prints the list of accounts", async (args, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (const account of accounts) {
+    console.log(account.address);
+  }
+});
+
+// You need to export an object to set up your config
+// Go to https://hardhat.org/config/ to learn more
+const config: HardhatUserConfig = {
+  solidity: "0.8.9",
+  networks: {
+    ropsten: {
+      url: "https://eth-ropsten.gateway.pokt.network/v1/lb/61d5f1ff1a94ee003a8e070c",
+      accounts: {
+        mnemonic:
+          "tube spin artefact salad slab lumber foot bitter wash reward vote cook",
```

Comment:
Is this also a problem?

Reply:
Same as before it should go into .env file but this is our test account and this mnemonic is in all the config.json.example files as well as airnode-examples

Reply:
Ah thanks, it did look vaguely familiar.